src/styles/main.scss

```diff
@@ -0,0 +1,11 @@
+
+@import "./utils/variables";
+@import "./reset.scss";
+@import "./blocks/clock.scss";
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
main.scss should contain only imports, class main  - its separate BEM block

Reply:
agree